### PR TITLE
google analytics

### PR DESCRIPTION
**Pull request by piotr-s-brainhub**
Deployed to https://beghp.github.io/gh-pages-rc-5<br><br>- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)<br>- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`<br><br>

### CODE REVIEW

File: docs-www/gatsby-config.js [hunk 1]
```diff
@@ -23,8 +23,23 @@ module.exports = {
...
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        trackingId: 'UA-62818184-6',
+        head: false,
+        anonymize: true,
+        respectDNT: true,
+        pageTransitionDelay: 0,
+        sampleRate: 5,
+        siteSpeedSampleRate: 10,
+        cookieDomain: 'brainhubeu.github.io',
```
Comment (RobertHebel) [inline docs-www/gatsby-config.js]: Why do we track only 5% for sampleRate and 10% of users for siteSpeedSampleRate?

Comment (piotr-s-brainhub) [reply docs-www/gatsby-config.js]: I thought these options from the docs would be good but I can remove them

Comment (piotr-s-brainhub) [reply docs-www/gatsby-config.js]: I removed them in https://github.com/brainhubeu/react-carousel/pull/342/commits/c91a5ce71cb9b9b59f5a65c168db392f46ba8b4d

Comment (RobertHebel) [reply docs-www/gatsby-config.js]: I'm just wondering what values would be optimal for our needs. According to [docs](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#create) the default value for Sample Rate is `100` which means 100% of users will be tracked. And for the `siteSpeedSampleRate` the default value is 10% of users

**Pull request merged**